docs/quickstart/access-control/aws-iam-eks.mdx

```diff
@@ -0,0 +1,330 @@
+For the `credentials-operator`, substitute `system:serviceaccount:otterize-system:intents-operator-controller-manager` with `system:serviceaccount:otterize-system:credentials-operator-controller-manager.
+
+
+3. Create IAM policies for these two roles.
+They should grant access to all `iam` permissions in the account, and a subset of `organizations` permissions. Here's an example:
+```yaml
+    Statement:
+        - Effect: Allow
+    Action:
+        - "iam:*"
+        - "organizations:DescribeAccount"
+        - "organizations:DescribeOrganization"
+        - "organizations:DescribeOrganizationalUnit"
+        - "organizations:DescribePolicy"
+        - "organizations:ListChildren"
+        - "organizations:ListParents"
+        - "organizations:ListPoliciesForTarget"
+        - "organizations:ListRoots"
+        - "organizations:ListPolicies"
+        - "organizations:ListTargetsForPolicy"
+    Resource: "*"
+```
+
+
+</details>
+
+## Tutorial
+
+### Create an S3 bucket for the server to use
+
+First, we need to pick an S3 bucket name. Because S3 buckets are globally unique, we will save the bucket name in an environment variable for use later.
+
+```bash
+export BUCKET_NAME=otterize-tutorial-bucket-`date +%s`
+echo $BUCKET_NAME
+```
+
+```bash
+aws s3api create-bucket --bucket $BUCKET_NAME --region us-west-2 --create-bucket-configuration LocationConstraint=us-west-2
+```
+
+### Deploy the sample server and client
+
+```shell
+kubectl create namespace otterize-tutorial-iam
+kubectl apply -n otterize-tutorial-iam -f ${ABSOLUTE_URL}/code-examples/aws-iam-eks/client-and-server.yaml
+kubectl patch deployment -n otterize-tutorial-iam server --type='json' -p="[{\"op\": \"replace\", \"path\": \"/spec/template/spec/containers/0/env\", \"value\": [{\"name\": \"BUCKET_NAME\", \"value\": \"$BUCKET_NAME\"}]}]"
+```
+
+<details>
+<summary>Expand to see the deployment YAML</summary>
+
+```yaml
+{@include: ../../../static/code-examples/aws-iam-eks/client-and-server.yaml}
+```
+
+</details>
+
+### View logs for the server - access denied
+The server logs will show that it fails to upload files to the S3 bucket.
+
+```bash
+kubectl logs -f -n otterize-tutorial-iam deploy/server
+```
+
+```bash
+2023/11/17 19:08:30 Couldn't upload file testfile.0.txt to otterize-tutorial-bucket-1700247949:testfile.0.txt.
+Here's why: operation error S3: PutObject, https response error StatusCode: 403, RequestID: W9ZBNWPDFEMTBBS5, HostID: gsOlyxAmcUqFiYKzB7IwBoPia/Z2TDHEoaJjXT/qAxdduD4H17XDoGu21Pb7zeMms/YeThv2mJk=,
+# highlight-next-line
+api error AccessDenied: Access Denied
+```
+
+### Annotate the server pod to create an AWS IAM role
+Annotate the server `Pod` so that the Otterize credentials operator creates an AWS IAM role for it and binds its Kubernetes ServiceAccount to the newly created role.
+
+To do this, we won't be annotating the `Pod` directly, but instead patching the `Deployment` we created earlier so that it updates the `Pod`.
```

Comment:
This is a bit confusing, since in the previous paragraph you said we *would* be annotating the `Pod`

Reply:
It is! But it's just how it works, so I think people won't be confused.